extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -677,8 +696,6 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
         contactItems.removeClass('active');
         $(this).addClass('active');
       });
-      this.view.S.cached('contacts').find('ul li.auth_contacts').click(this.view.setHandler(() =>
-        this.authContacts(this.view.acctEmail), this.view.errModule.handle(`authorize contact search`)));
```

Comment:
Was this functionality replicated elsewhere? If it wasn't, and tests still pass, then we need to write a test for this (unless this was somehow already duplicate?)